src/components/LocationPage/ShareButtons.tsx

```diff
@@ -1,33 +1,30 @@
 import React, { useState } from 'react';
 import urlJoin from 'url-join';
 import SocialButtons from './SocialButtons';
-import {
-  SaveOrShareContainer,
-  DesktopButtonsWrapper,
-  MobileButtonsWrapper,
-  ClickAwayWrapper,
-  SocialButtonsWrapper,
-} from './ShareButtons.style';
+import { Wrapper } from './ShareButtons.style';
 import ShareButton from 'components/NewLocationPage/ShareButton/ShareButton';
 import { ClickAwayListener } from '@material-ui/core';
 import makeChartShareQuote from 'common/utils/makeChartShareQuote';
 import * as urls from 'common/urls';
 import { Region } from 'common/regions';
 
-interface InnerContentProps {
+const ShareButtons: React.FC<{
```

Comment:
this file had 2 components: a main (`ShareButtons`) and an inner component (`InnerContent`). did a bunch of cleanup, deleted some unnecessary wrappers, and combined the components into a single one (`ShareButtons`)